modules/ROOT/pages/browser-settings.adoc

```diff
@@ -0,0 +1,24 @@
+= Configure Studio's Browser Settings
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio comes bundled with a default browser to display elements of the Anypoint Platform UI. For example, when you deploy to CloudHub from Studio, the Runtime Manager UI displayed in Studio is rendered using a built-in browser.
+
+It is recommended to use the built-in browser, however, if the default browser fails to display you can configure Studio to your system's native browser to render the Anypoint Platform UI:
+
+. Navigate to *Preferences* > *Anypoint Studio* > *Browser*
+. Select the type of default browser for Studio:
++
+image::browser-configuration.png[]
+<1> *Built-in*: Uses the built-in browser. This is the recommended setting.
+<2> *Native*: Uses the native browser of your operative system.
+. Click *Preview* to test the browser's configuration. +
+If the page previews correctly, then Studio's browser is correctly configured.
```

Comment:
Do we need this sentence if we change the one above it?
Also, I can see a possibility that my suggestion could have subtly changed meaning (verifying the config change vs verifying the resulting display), so please use your best judgment.